src/main/java/org/cactoos/Func.java

```diff
@@ -62,6 +62,9 @@
      * @param <X> Type of input
      * @param <Y> Type of output
      * @since 0.10
+     * @todo #852:30min Move NoNulls implementations of Func, BiFunc, Proc, and
+     *  BiProc to their own packages in order to provide the NoNulls classes API
```

Comment:
@dgroup you've written "packages" in all these puzzles when in fact it should be "files". Can you please correct this typo in order to avoid confusion?